### DISCUSSION TITLE
Using sh1mmer to enroll a device that hasn't ever been enrolled

### DISCUSSION THREAD

**Discussion by twlswan** [3kh0/ext-remover#477] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
Title. I know you can re-enroll devices  but can you enroll a "random" chromebook and make it enterprise-managed? It would be straight up essential to get a freaking copy of parallels. Thanks.

**Comment by amelia-bartlett** [comment]


**Reply by twlswan** [reply]
well.. the normal enrollment procedure needs your org info and since I don't have them I can't go beyond that point.
so it's either sh1mmer or nothing for me.

**Reply by ehunt34** [reply]
Sh1mmer wouldn't do anything for you. The only thing it does is change the check_enrollment flag in the Vendor Product Data (VPD). The flag basically controls whether or not your Chromebook phones home to Google's servers. Even if it was on, the Chromebook Serial Number wouldn't be set as enrolled so it would work as a personal Chromebook, as it should. In order to enroll that Serial Number, you need to enroll it in a domain, which only your school is able to do.

**Reply by twlswan** [reply]
> Sh1mmer wouldn't do anything for you. The only thing it does is change the check_enrollment flag in the Vendor Product Data (VPD). The flag basically controls whether or not your Chromebook phones home to Google's servers. Even if it was on, the Chromebook Serial Number wouldn't be set as enrolled so it would work as a personal Chromebook, as it should. In order to enroll that Serial Number, you need to enroll it in a domain, which only your school is able to do.

Yeah I was expecting the actual enrollment to be something server-side. thanks for the clarification.

**Reply by amelia-bartlett** [reply]
> well.. the normal enrollment procedure needs your org info and since I don't have them I can't go beyond that point. so it's either sh1mmer or nothing for me.

if you are enrolling, why don't you have the org info?  surely you could ask the org to give it to you?

i don't know what your use case is for this anyway, if you are meant to be enrolling chromebooks to an org, you should ask them for a quick primer.  if you arent meant to be enrolling them, then i cant help you and even if i could, i don't want to.

**Reply by twlswan** [reply]
> > well.. the normal enrollment procedure needs your org info and since I don't have them I can't go beyond that point. so it's either sh1mmer or nothing for me.
> 
> if you are enrolling, why don't you have the org info? surely you could ask the org to give it to you?
> 
> i don't know what your use case is for this anyway, if you are meant to be enrolling chromebooks to an org, you should ask them for a quick primer. if you arent meant to be enrolling them, then i cant help you and even if i could, i don't want to.

I'm merely a curious high schooler, I have nothing of the sort, my school doesn't even know what chromebooks are:)
my chromebook is a personal device. This seemed like a nice way to finally get my hands on one enterprise/educational exclusive feature I wanted for a long time, which is being eligible to buy a Parallels license after years of pointlessly waiting for them to make it available to everyone.

**Reply by S-PScripts** [reply]
seems like you can't enroll a device thats never been enrolled

**Reply by schoolexploitkid** [reply]
> seems like you can't enroll a device thats never been enrolled

yes, there is such a thing called "enterprise" chromebooks, and they are sold seperately from the normal ones.

**Reply by schoolexploitkid** [reply]
i found this when searching for leaked shims

**Reply by S-PScripts** [reply]
> > seems like you can't enroll a device thats never been enrolled
> 
> yes, there is such a thing called "enterprise" chromebooks, and they are sold seperately from the normal ones.

definitely, i got a chromebook brand new with the box unwrapped (sticker was still on) that school gave to me. it was enterprised
